modules/tiddit/sv/main.nf

```diff
@@ -2,53 +2,52 @@ process TIDDIT_SV {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::tiddit=2.12.1" : null)
+    conda (params.enable_conda ? "bioconda::tiddit=3.0.0" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/tiddit:2.12.1--py38h1773678_0' :
-        'quay.io/biocontainers/tiddit:2.12.1--py38h1773678_0' }"
+        'https://depot.galaxyproject.org/singularity/tiddit:3.0.0--py39h59fae87_1' :
+        'quay.io/biocontainers/tiddit:3.0.0--py39h59fae87_1' }"
 
     input:
-    tuple val(meta), path(bam)
+    tuple val(meta), path(input), path(index)
     path  fasta
-    path  fai
+    path  bwa_index
 
     output:
-    tuple val(meta), path("*.vcf")        , emit: vcf
-    tuple val(meta), path("*.ploidy.tab") , emit: ploidy
-    tuple val(meta), path("*.signals.tab"), emit: signals
-    path  "versions.yml"                  , emit: versions
+    tuple val(meta), path("*.vcf")         , emit: vcf
+    tuple val(meta), path("*.ploidies.tab"), emit: ploidy
+    path  "versions.yml"                   , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def reference = fasta ? "--ref $fasta" : ""
     """
+    [[ -d $bwa_index ]] && for i in `ls $bwa_index`; do [[ -f $fasta && ! "\$i" =~ .*"$fasta".* ]] && ln -s $bwa_index/\$i ${fasta}.\${i##*.} || ln -s $bwa_index/\$i \$i; done
```

Comment:
```suggestion
    [[ -d $bwa_index ]] && for i in $bwa_index/*; do [[ -f $fasta && ! "\$i" =~ .*"$fasta".* ]] && ln -s $bwa_index/\$i ${fasta}.\${i##*.} || ln -s $bwa_index/\$i \$i; done
```
Need to double check the iterator is correct, but usually one doesn't need to use `ls` for this.